.env.local.sample

```diff
@@ -1,21 +0,0 @@
-NEXT_PUBLIC_DISCORD_CLIENT_ID=xxx
```

Comment:
Please don't touch this file.
Eqverytime, before requesting a review, please check here https://github.com/nmalzieu/starky/pull/77 that you didn't do anything wrong